rb/lib/selenium/webdriver/edge/legacy_support.rb

```diff
@@ -66,17 +64,18 @@ def mouseUp
         end
 
         def mouseMoveTo(element, x = nil, y = nil)
-          params = { :element => element }
+          params = { element: element }
 
           if x && y
-            params.merge! :xoffset => x, :yoffset => y
+            params[:xoffset] = x
+            params[:yoffset] = y
           end
 
           execute :mouseMoveTo, {}, params
         end
 
         def sendKeysToActiveElement(key)
-          execute :sendKeysToActiveElement, {}, :value => key
+          execute :sendKeysToActiveElement, {}, {value: key}
```

Comment:
Also, some hashes have space after `{` and before `}`, but this one does not. Let's come up with one consistent approach (I prefer to avoid spaces to distinguish with blocks).